workflow/rules/long_read.smk

```diff
@@ -56,7 +56,7 @@ rule nanofilt:
 
 rule downsample_and_trim_raw:
     input:
-        primer=config["preprocessing"]["amplicon-primers"],
+        primer=config["preprocessing"]["amplicon-primers"]["ont"],
```

Comment:
What if no primers are used? This will fail with a key error then.

Reply:
illumina + amplicon exclusive